docs/get-started/roles-and-permissions/index.md

```diff
@@ -0,0 +1,10 @@
+# Roles and permissions in PMM
+
+
+Roles are the sets of permissions and configurations that determine which metrics a user can access.
+
+Each PMM user is associated with a role that includes permissions. Permissions determine the privileges that a user has in PMM.  For example, the Admin role allows an administrator to create and delete users.
```

Comment:
I think this might confuse users, the Grafana role Admin allows the administrators to create/delete users. The Access Roles aren't connected to Grafana roles.

Reply:
Fixed.